src/products/mozilla_org/testing.md

```diff
@@ -9,7 +9,7 @@ Mozilla.org is highly visible because the site houses the basic info of all Mozi
 * Repository: https://github.com/mozilla-l10n/www.mozilla.org/
 * Pontoon: https://pontoon.mozilla.org/projects/mozillaorg/
 
-It’s highly advised you to ask other community members to conduct peer review not only on Pontoon, but on staging. While not all the languages are required for certain projects, each community can opt in the projects at a later time.
+It’s highly advised you to ask other community members to conduct peer review not only on Pontoon, but on production. While not all the languages are required for certain projects, each community can opt in the projects at a later time.
```

Comment:
Why are we suggesting to test in production? Also, I realize it's not super clear to me what a project is in the context of mozilla.org?

Reply:
I have changed to staging and production. The only meaningful way to test on stage is when a page is launched on a particular day, not as soon as it is localized. 
As for what is a project or product, technically it is a product, as it doesn't  end when the deadline has gone.

Reply:
OK, i didn't realize you were talking about external projects/products